bodhi/tests/server/test_utils.py

```diff
@@ -70,3 +73,59 @@ def test_sorted_builds(self):
         b1, b2 = sorted_builds([new, old])
         assert b1 == new, b1
         assert b2 == old, b2
+
+
+class TestCMDFunctions(unittest.TestCase):
+    @mock.patch('bodhi.server.log.info')
```

Comment:
Again apologies about my confusion leading you astray here - let's also mock the debug log and assert that the right things are getting logged at debug in each of these three cases.